Integrations/integration-RsaNetWitnessPacketsAndLogs.yml

```diff
@@ -344,11 +439,95 @@ script:
         case 'nw-decoder-parsers-upload':
             var res = decoderImport('/decoder/parsers/upload');
             return res;
+        case  'netwitness-msearch':
+            return buildMdAndContext(doReq('GET', commandToPath[command], args, args.responseType))
+        case  'netwitness-search':
+            return buildMdAndContext(doReq('GET', commandToPath[command], args, args.responseType))
+        case  'netwitness-query':
+            return buildMdAndContext(doReq('GET', commandToPath[command], args, args.responseType))
+        case 'netwitness-packets':
+            return parseDownloadResponse(doReq('GET', commandToPath[command], args, args.responseType));
         default:
             return parseResponse(doReq('GET', commandToPath[command], args, args.responseType));
     }
   type: javascript
   commands:
+  - name: netwitness-msearch
```

Comment:
doesn't seem you have added outputs and important to any of the commands
this is important so it can be viewed and used inside playbooks